src/widgets/DataTable/DataTableRow.js

```diff
@@ -263,11 +270,19 @@ const DataTableRow = React.memo(
               );
             }
 
-            case COLUMN_TYPES.DROP_DOWN:
+            case COLUMN_TYPES.DROP_DOWN: {
+              const { isVaccine = false } = rowData ?? {};
+
+              const disabledConditions = {
+                [COLUMN_KEYS.CURRENT_VVM_STATUS]: !isVaccine,
+              };
+
+              const disabledVVMStatus = disabledConditions[columnKey] || isDisabled;
```

Comment:
This ` || isDisabled` isn't necessary as it is checked when used in line `285`. I prefer splitting it out - and having `disabledVVMStatus` specific to VVM status

Reply:
Thanks! 6701530